includes/classes/InitSystem.php

```diff
@@ -0,0 +1,284 @@
+            $result = 'SUCCESS';
+            $this->actionList[] = ['type' => 'include', 'filePath' => $filePath];
+        }
+        $this->debugList[] = 'loading include - ' . $filePath . ' - ' . $result;
+    }
+
+    protected function processAutoTypeInit_script($entry)
+    {
+        $actualDir = DIR_WS_INCLUDES . 'init_includes/';
+        if ($entry['loaderType'] == 'plugin') {
+            $actualDir = $this->findPluginDirectory($actualDir, $entry['pluginInfo']['unique_key']);
+        }
+        if (file_exists($actualDir . 'overrides/' . $entry['loadFile'])) {
+            $actualDir = $actualDir . 'overrides/';
+        }
+        $this->actionList[] = ['type' => 'require', 'filePath' => $actualDir . $entry['loadFile']];
+        $this->debugList[] = 'loading init_script - ' . $actualDir . $entry['loadFile'];
+
+    }
+
+    protected function loadAutoLoadersFromSystem($loaderType, $rootDir, $plugin = [])
+    {
+        $fileList = $this->fileSystem->listFilesFromDirectory($rootDir, '~^[^\._].*\.php$~i');
+        $fileList = $this->processForOverrides($fileList, $rootDir);
+        $loaderList = $this->getLoadersFromFileList($fileList);
+        $loaderList = $this->processLoaderListForType($loaderType, $loaderList, $plugin);
+        return $loaderList;
+    }
+
+    protected function loadPluginAutoLoaders($loaderType)
+    {
+        $pluginLoaderList = [];
+        foreach ($this->installedPlugins as $plugin) {
+            $baseDir = $this->pluginManager->getPluginVersionDirectory($plugin['unique_key'], $this->installedPlugins);
+            $rootDir = $baseDir . $this->context . '/includes/auto_loaders';
+            $loaderList = $this->loadAutoLoadersFromSystem($loaderType, $rootDir, $plugin);
+            $pluginLoaderList = $this->mergeAutoLoaders($pluginLoaderList, $loaderList);
+        }
+        return $pluginLoaderList;
+    }
+
+    protected function processForOverrides($fileList, $rootDir)
+    {
+        $newFileList = [];
+        $baseDir = $rootDir;
+        $overrideDir = $baseDir . '/overrides';
+        foreach ($fileList as $file) {
+            if (!$this->fileMatchesLoaderPrefix($file)) continue;
+            $filePath = $baseDir . '/' . $file;
+            if ($this->overrideFileExists($file, $overrideDir)) $filePath = $overrideDir . '/' . $file;
+            $newFileList[] = $filePath;
+        }
+        return $newFileList;
+    }
+
+    protected function fileMatchesLoaderPrefix($file)
+    {
+        $fileParts = explode('.', $file);
+        if ($fileParts[0] !== $this->loaderPrefix) return false;
+        return true;
+    }
+
+    protected function overrideFileExists($file, $overrideDir)
+    {
+        if (file_exists($overrideDir . '/' . $file)) return true;
+        return false;
+    }
+
+    protected function getLoadersFromFilelist($fileList)
+    {
+        foreach ($fileList as $file) {
+            require($file);
+        }
+        return $autoLoadConfig;
```

Comment:
@zcwilt 
This `$autoLoadConfig` seems to be the wrong variable. 

But returning `$fileList` instead is not a correct solution.

Reply:
$autoLoadConfig is defined in the auto_loaders file that is being required